components/builder-db/src/migrations/2018-11-05-111111_diesel_query_improvements/up.sql

```diff
@@ -0,0 +1,93 @@
+DROP FUNCTION IF EXISTS get_account_by_name_v1(text);
```

Comment:
Would be good to have a related PR for on-prem to ensure we don't break the DB merge migration with schema changes